fixtures/additional-modules/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import dep from "./dep";
+import text from "./text.txt";
+
+export default <ExportedHandler>{
+	async fetch(request) {
+		const url = new URL(request.url);
+		if (url.pathname === "/dep") {
+			return new Response(dep);
+		}
+		if (url.pathname === "/text") {
+			return new Response(text);
+		}
+		if (url.pathname === "/dynamic") {
+			return new Response((await import("./dynamic.js")).default);
+		}
+		if (url.pathname.startsWith("/lang/")) {
+			const language = url.pathname.substring("/lang/".length);
+			return new Response(
+				(await import(`./lang/${language}.js`)).default.hello
```

Comment:
```suggestion
			const identifier = "./dynamic.js";
			return new Response((await import(identifier)).default);
		}
		if (url.pathname.startsWith("/lang/")) {
			const language = url.pathname.substring("/lang/".length);
			const identifier = `./lang/${language}.js`;
			return new Response(
				(await import(identifier)).default.hello
```

Splitting the identifiers out like this is just a way to ensure into the future that these things don't get bundled by esbuild (and therefore that these dynamic imports are actually still working, in the event that we do update esbuild beyond 0.19.0 and somehow mess up the opt-out of collapsing these dynamic imports (though hopefully we do add tests at that time to assert that doesn't happen anyway!)

https://github.com/evanw/esbuild/issues/3337
https://esbuild.github.io/api/#non-analyzable-imports